components/IndicatorStat/utils/data.ts

```diff
@@ -163,7 +172,7 @@ export const getIndicatorsValue = (
     });
     const aggregate = aggregateProcessedData(values, options);
     if (typeof aggregate === 'number') {
-      return addPrefixAndSuffix(formatNumber(aggregate), options);
+      return addPrefixAndSuffix(formatNumber(aggregate, typeof decimalCount == 'number' ? decimalCount : 1), options);
```

Comment:
`options.decimalCount` which should always be a number or undefined ... if undefined, the default is being handled within `formatNumber`.

Reply:
Updated this as well